### ISSUE TITLE
Adjust the productinfo UI to the changed rest URL scheme

### ISSUE THREAD

**Issue by lpuskas** [banzaicloud/productinfo#49]
The productinfo REST api has been normalized, changes need to be reflected in the UI code, so that the appropriate calls are executed against the proper endpoints.

The new scheme has the form:
```
/api/v1/providers
/api/v1/providers/:provider
/api/v1/providers/:provider/services
/api/v1/providers/:provider/services/:service/regions
/api/v1/providers/:provider/services/:service/regions/:region
/api/v1/providers/:provider/services/:service/regions/:region/images
/api/v1/providers/:provider/services/:service/regions/:region/images?product=productID
/api/v1/providers/:provider/services/:service/regions/:region/products
/api/v1/providers/:provider/services/:service/regions/:region/products/:attribute
/api/v1/providers/:provider/services/:service/regions/:region/products/:attribute?image=imageID
```

On the UI the "provider" dropdown needs to be populated with data in the following form:

```provider1
  - service1
  - service2
provider1
  - service1
  - service2
................
```

The data in the dropdown may be retrieved with the call

`curl -L -X GET 'http://localhost:9090/productinfo/api/v1/providers'`

**Comment by lpuskas** [comment]
To run the productinfo locally (provided docker-compose / docker are installed and aws credentials are available) execute:

`make pi-start`

To stop the app execute:

`make pi-stop`

**Comment by lpuskas** [comment]
fixed by: #14b524935f71bb79ee7bbdc75bd73838c0e31e2b
